.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow creates a release PR if changesets is existing
+# When the changesets PR is merged, it will create tags to Github and publish packages.
+name: Create Release Pull Request
+
+on:
+  push:
+    branches:
+      - master
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  create-release-pr:
+    env:
+      NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
It should be unnecessary because frolint's dependencies are public (as frolint itself is public). Note that NPM_TOKEN is also specified below separately for writing.